src/main/java/ch/wisv/areafiftylan/service/TicketServiceImpl.java

```diff
@@ -45,6 +46,11 @@ public TicketServiceImpl(TicketRepository ticketRepository, UserService userServ
         this.teamService = teamService;
     }
 
+    @Autowired
+    public void setRFIDService(RFIDService rfidService){
```

Comment:
This had to be implemented due to a circular dependency between RFIDService and TicketService. Ticket Transfer is dependent on RFID, RFID is dependent on tickets. A solution to remove the circular dependency would be to extract the Ticket Transfer logic to a seperate service. I invite you to continue the discussion in #250.

See [StackOverflow](http://stackoverflow.com/questions/5632596/understanding-spring-context-initialization-order) for motivation.